ee/clickhouse/queries/experiments/test_utils.py

```diff
@@ -54,7 +54,11 @@ def test_with_feature_flag_properties_on_events(self):
 
     def test_with_no_feature_flag_properties_on_actions(self):
         action_credit_card = Action.objects.create(team=self.team, name="paid")
-        ActionStep.objects.create(action=action_credit_card, event="paid", properties=[{"$os": "Windows"}])
+        ActionStep.objects.create(
+            action=action_credit_card,
+            event="paid",
+            properties=[{"key": "$os", "type": "event", "value": ["Windows"], "operator": "exact"}],
```

Comment:
I checked and we're not using this oldschool style of properties anywhere, except these tests.

Reply:
I ran a query looking for such fields in the dashboard item filters, and found nothing. This was a while ago, so I lost the query 😬. Can try running again and posting results here.